pyEpiabm/pyEpiabm/sweep/travel_sweep.py

```diff
@@ -0,0 +1,239 @@
+        self.remove_leaving_individuals(time)
+
+    def create_introduced_idividuals(self, time,
+                                     number_individuals_introduced):
+        """
+        Create individuals and assign them an age and infectious status. This
+        is done based on age proportions if age is used in the model.
+        Individuals are assigned an age between 15-80 years and are all
+        infected (mild or asymptomatic).
+
+        Parameters
+        ----------
+        time : float
+            Simulation time
+        number_individuals_introduced: int
+            Infected individuals added to population at certain time step
+        """
+        asymp_prop = Parameters.instance().host_progression_lists[
+            "prob_exposed_to_asympt"]
+        if Parameters.instance().use_ages:
+            # Age used in model
+            age_prop = Parameters.instance().age_proportions
+            # travellers are between 15-80 years
+            age_prop_adjusted = [0.0 if i in [0, 1, 2, 16] else prop for
+                                 i, prop in enumerate(age_prop)]
+            w = age_prop_adjusted / sum(age_prop_adjusted)
+            microcell_split = np.random.multinomial(
+                number_individuals_introduced, w, size=1)[0]
+            for age in range(len(age_prop_adjusted)):
+                number_indiv_agegroup = microcell_split[age]
+                number_indiv_agegroup_InfectedAsympt = \
+                    math.floor(asymp_prop[age] * number_indiv_agegroup)
+                number_indiv_agegroup_InfectedMild = \
+                    number_indiv_agegroup - \
+                    number_indiv_agegroup_InfectedAsympt
+                self.initial_microcell.add_people(
+                    number_indiv_agegroup_InfectedAsympt,
+                    status=InfectionStatus.InfectASympt,
+                    age_group=age)
+                self.initial_microcell.add_people(
+                    number_indiv_agegroup_InfectedMild,
+                    status=InfectionStatus.InfectMild,
+                    age_group=age)
+        else:
+            # Age not used in model
+            if isinstance(asymp_prop, list):
+                asymp_prop = asymp_prop[0]
+            number_indiv_InfectedAsympt = \
+                math.floor(asymp_prop * number_individuals_introduced)
+            number_indiv_InfectedMild = number_individuals_introduced - \
+                number_indiv_InfectedAsympt
+            self.initial_microcell.add_people(
+                    number_indiv_InfectedAsympt,
+                    status=InfectionStatus.InfectASympt)
+            self.initial_microcell.add_people(
+                    number_indiv_InfectedMild,
+                    status=InfectionStatus.InfectMild)
+
+        for person in self.initial_cell.persons:
+            # Assign introduced individuals a time to stay
+            person.travel_end_time = time + random.randint(2, 14)
```

Comment:
Should these be taken from the parameters file and then read in, rather than hardcoded?

Reply:
I assume you mean the (2,14)? I will make them a parameter: `'duration_travel_stay': [2, 14]`

Reply:
Yep that looks great - best to test the duration is lasting as expected and also document the new parameter in the wiki